src/modules/QtGraphicalEffects/RectangularGlow.js

```diff
@@ -0,0 +1,95 @@
+QmlWeb.registerQmlType({
+  module: "QtQuick",
+  name: "RectangularGlow",
+  versions: /.*/,
+  baseClass: "Item",
+  properties: {
+    cached: {
+      type: "bool",
+      initialValue: false
```

Comment:
Techically, `initialValue` is not required here and below (except for the `color`) — if `initialValue` is not specified, a type-specific default is used instead (like in QML). For `bool`, that is `false`, for numeric — `0`, for `color` — black.

It's ok to include those, though, so you can ignore this comment.